index.js

```diff
@@ -5,9 +5,13 @@ exports.decompress = decompress;
 
 var brotli  = require('iltorb');
 var through = require('through2');
+var extend = require('extend');
 
 function compress(params) {
-  params = params || {};
+  params = extend(true, {
```

Comment:
I don't think this is complex enough to warrant an extra dependency.

We can just use this here and in `decompress` as well.
```js
function getExtension(params) {
  return params && params.extension || 'br';
}

// ...

var extension = getExtension(params);
```